crates/bevy_transform/src/systems.rs

```diff
@@ -19,6 +20,21 @@ pub fn sync_simple_transforms(
         *global_transform = GlobalTransform::from(*transform);
     });
 }
+/// Update [`GlobalTransform`] component of entities that aren't in the hierarchy
+/// and lost their parent last frame.
+///
+/// Third party plugins should use [`transform_propagate_system_set`](crate::transform_propagate_system_set)
+/// to propagate transforms correctly.
+pub fn sync_simple_transforms_orphaned(
```

Comment:
This should be added in `transform_propagate_system_set` for third party plugin use.